pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -194,7 +197,7 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 	// Reuse LimitRanger logic to apply limit/req defaults from the project. Ignore validation
 	// errors, assume that LimitRanger will run after this plugin to validate.
 	glog.V(5).Infof("%s: initial pod limits are: %#v", api.PluginName, pod.Spec)
-	if err := a.LimitRanger.Admit(attr); err != nil {
+	if err := a.LimitRanger.(admission.MutationInterface).Admit(attr); err != nil {
```

Comment:
don't we also need to implement Validate() and delegate to a.LimitRanger.(...).Validate()?

Reply:
> don't we also need to implement Validate() and delegate to a.LimitRanger.(...).Validate()?

I don't think so.  This wanted the mutation half, but the pre-existing comments indicated that it counted on the stock limitranger running after.